plugins/filter_aws/aws.c

```diff
@@ -1048,6 +1116,12 @@ static struct flb_config_map config_map[] = {
      "if both tags_include and tags_exclude are specified, configuration is invalid"
      " and plugin fails"
     },
+    {
+     FLB_CONFIG_MAP_INT, "tags_retry_interval_s", "300",
```

Comment:
Do we want to call this new configuration option `tags_retry_interval_s`? In case we add retries for more metadata groups in the future, we might consider using just one `retry_interval_s` and set the retries for all groups via just one option.

What do you think about changing the name of this option to `retry_interval_s`?

Reply:
Yea I think this makes more sense as the starting point for retries. I had meant to ask why it was tag specific but it seems I got distracted and forgot to put that comment.

Reply:
OK, changed: https://github.com/fluent/fluent-bit/pull/7245/commits/deff300b717514818154eb8bb1eb05b40508ea8f